.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description of PR purpose/changes
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
Can we also have a link to the Jira ticket? It is so much easier just to click on a link than to have to find the ticket in Jira manually.

Reply:
@ialarmedalien I added that at the very end. Where would you like this added?

Reply:
You could call it out in a section of it's own i.e. ## JIRA Story/Task, and have it in its own section so it is easy to find.

Reply:
@eamahanna Done